_data/_samples.json

```diff
@@ -105,13 +106,11 @@
       "title": "'Hello, blinky!'",
       "description": "A 'Hello, world' sample to control an LED from a breadboard",
       "link": [
-        "samples/HelloBlinky.htm",
-        "samples/HelloBlinkyNode.htm"
+        "samples/HelloBlinky.htm"

```

Comment:
If we're going to a single link/language per sample, please remove them from the json array - see other examples in this file like Alljoyn.js